From 1662ea7a33eb8878df9561e79c251bc1a57f2671 Mon Sep 17 00:00:00 2001
From: Naushir Patuck <naush@raspberrypi.com>
Date: Fri, 20 May 2022 11:58:21 +0100
Subject: [PATCH] configs: Enable the IMX296 device module in platform
 defconfig

Build this module by default for the following:
bcm2711_defconfig
bcm2709_defconfig
bcmrpi3_defconfig
bcmrpi_defconfig

Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 3615a53cef18..16698a5be3b9 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -928,6 +928,7 @@ CONFIG_VIDEO_ARDUCAM_PIVARIETY=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_IMX258=m
 CONFIG_VIDEO_IMX290=m
+CONFIG_VIDEO_IMX296=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_IMX519=m
 CONFIG_VIDEO_OV2311=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index 923eb2a0d641..e9da4fda5c40 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -944,6 +944,7 @@ CONFIG_VIDEO_ARDUCAM_PIVARIETY=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_IMX258=m
 CONFIG_VIDEO_IMX290=m
+CONFIG_VIDEO_IMX296=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_IMX519=m
 CONFIG_VIDEO_OV2311=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index d853915dad64..a0ecfe93a6ab 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -921,6 +921,7 @@ CONFIG_VIDEO_ARDUCAM_PIVARIETY=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_IMX258=m
 CONFIG_VIDEO_IMX290=m
+CONFIG_VIDEO_IMX296=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_IMX519=m
 CONFIG_VIDEO_OV2311=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index f8a0aee199c3..410e2f7c9dfa 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -950,6 +950,7 @@ CONFIG_VIDEO_ARDUCAM_PIVARIETY=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_IMX258=m
 CONFIG_VIDEO_IMX290=m
+CONFIG_VIDEO_IMX296=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_IMX519=m
 CONFIG_VIDEO_OV2311=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 95885c250f30..e456d05c8116 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -892,6 +892,7 @@ CONFIG_VIDEO_ARDUCAM_PIVARIETY=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_IMX258=m
 CONFIG_VIDEO_IMX290=m
+CONFIG_VIDEO_IMX296=m
 CONFIG_VIDEO_IMX477=m
 CONFIG_VIDEO_IMX519=m
 CONFIG_VIDEO_OV2311=m
-- 
2.32.0

